app/src/main/java/org/fossasia/susi/ai/activities/MainActivity.java

```diff
@@ -276,7 +276,12 @@ private void parseSusiResponse(SusiResponse susiResponse, int i) {
 
         switch(actionType) {
             case Constant.ANCHOR :
-                // TODO : Add code for anchor here
+                try {
+                    answer = "<a href=\""  +susiResponse.getAnswers().get(0).getActions().get(i).getAnchorLink() + "\">" + susiResponse.getAnswers().get(0).getActions().get(1).getAnchorText() + "</a>";
+                } catch (Exception e) {
+                    Log.d(TAG, e.getLocalizedMessage());
```

Comment:
Add `answer = getString(R.string.error_occurred_try_again);` after log statement. So, that the answer doesn't remain null if an there's an exception. Also squash the commits. Rest is fine.